src/pages/Selector/index.jsx

```diff
@@ -37,6 +37,10 @@ const Selector = ({ employees, templates }) => (
       />
       <GenerateCVButton />
     </div>
+    <Modal isOpen title="Preview Document" bodyText={mockDataForModal.bodyText}>
+      <CancelModalButton className="cancel-button" />
+      <DownloadModalButton className="download-button" />
```

Comment:
As mentioned above about styled components. Is the DownloadModalButton a styled component?

Reply:
I've removed these